recipe-client-addon/bin/tc/test.sh

```diff
@@ -4,20 +4,14 @@ set -eu
 # mach wants this
 export SHELL=$(which bash)
 
-# Creates gecko-dev-master
-echo 'Downloading gecko-dev...'
-curl -sL https://github.com/mozilla/gecko-dev/archive/master.tar.gz | tar xz
-
-pushd normandy/recipe-client-addon
-npm install
-./bin/update-mozilla-central.sh ../gecko-dev/
-popd
+# creates gecko-dev-master
+echo 'Downloading build result'
+curl -sfL "$BUILD_RESULT" | tar xz
 
```

Comment:
Log messages declaring that we're running lints and tests like the other ones you added would be neat.